components/molecules/icon-button/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Button from '_components/atoms/button'
+import Icon from '_components/atoms/icon'
+
+const IconButton = ({ onClick, disabled, className, iconURL, iconDescription }) => (
```

Comment:
Wouldn't we remove it?

Reply:
We removed the close-icon component.
I think it's good to have an IconButton component, what do you think?